pages/releases.js

```diff
@@ -21,7 +21,13 @@ const ReleaseAnchor = styled(Anchor)`
 `;
 
 const Releases = ({ releases, sidebarPages }) => (
-  <DocsLayout useDocsSidebarMenu={false} pages={sidebarPages} title="Releases" description="Styled Components Releases">
+  <DocsLayout
+    useDocsSidebarMenu={false}
+    pages={sidebarPages}
+    title="Releases"
+    description="Styled Components Releases"
+    feedLink={getReleasesAtomFeedURI()}
```

Comment:
Can you directly embed this functionality inside `DocsLayout` instead of composing the feed link here?